src/function/aggregate/distributive/minmax.cpp

```diff
@@ -522,7 +522,7 @@ static AggregateFunction GetMinMaxFunction(const LogicalType &type) {
 template <class OP, class OP_STRING, class OP_VECTOR>
 static void AddMinMaxOperator(AggregateFunctionSet &set) {
 	for (auto &type : LogicalType::ALL_TYPES) {
-		if (type.id() == LogicalTypeId::VARCHAR || type.id() == LogicalTypeId::BLOB) {
+		if (type.id() == LogicalTypeId::VARCHAR || type.id() == LogicalTypeId::BLOB || type.id() == LogicalTypeId::UUID) {
```

Comment:
UUID should not be here, no? UUID can be handled in min/max the same as hugeint.